doc/mgr/prometheus.rst

```diff
@@ -31,7 +31,7 @@ Configuration
 
 By default the module will accept HTTP requests on port ``9283`` on all IPv4
 and IPv6 addresses on the host.  The port and listen address are both
-configurable with ``ceph config-key set``, with keys
+configurable with ``ceph config set``, with keys
```

Comment:
i think both the document and the command sample are wrong. because the prometheus mgr module is using `get_localized_module_option` for retrieving the localized settings of `server_addr` and `server_port`. so, probably should revise it like

```
::

    ceph config set mgr mgr/prometheus/<mgr-id>/server_port
    ceph config set mgr mgr/prometheus/<mgr-id>/server_port
```

Reply:
OK, will change them accordingly.